lib/bugsnag/logger.ex

```diff
@@ -2,39 +2,41 @@ defmodule Bugsnag.Logger do
   require Bugsnag
   require Logger
 
-  use GenEvent
+  @behaviour :gen_event
```

Comment:
Just to confirm, since I don't have a great understanding of this myself... the `GenEvent` docs say:

> If your use case requires exactly what GenEvent provided, or you have to integrate with an existing `:gen_event`-based system, you can still use the `:gen_event` Erlang module.

This offers two possible reasons to use the `:gen_event` module — which one is true for us?

Reply:
Well, its the default way to implement what we need (error reporting).
Also, i've simply looked at all other libs and they all do that.

Reply:
Hm, I'm still not sure I understand which of those reasons applies to us. But if this seems to be the way other libraries are handling it, we're probably good. 👍

Reply:
It's the second reason: [Erlang's `error_logger` uses the gen_event behaviour for pluggable log handlers](http://erlang.org/doc/man/error_logger.html). 

Relevant line in this project: https://github.com/jarednorman/bugsnag-elixir/blob/master/lib/bugsnag.ex#L17